plans/install.pp

```diff
@@ -17,14 +17,14 @@
 plan peadm::install (
   # Standard
   Peadm::SingleTargetSpec           $primary_host,
-  Optional[Peadm::SingleTargetSpec] $primary_replica_host = undef,
+  Optional[Peadm::SingleTargetSpec] $replica_host = undef,
 
   # Large
   Optional[TargetSpec]              $compiler_hosts = undef,
 
   # Extra Large
-  Optional[Peadm::SingleTargetSpec] $puppetdb_database_host         = undef,
-  Optional[Peadm::SingleTargetSpec] $puppetdb_database_replica_host = undef,
+  Optional[Peadm::SingleTargetSpec] $primary_postgresql_host         = undef,
```

Comment:
May just be a matter of opinion but could be more level indentation here

Reply:
Totally agree and fixed.